invertedai/large/initialize.py

```diff
@@ -274,29 +240,112 @@ def get_number_of_agents_per_region_by_drivable_area(
 
     return filtered_regions
 
+@validate_call
+def _insert_agents_into_nearest_regions(
+    regions: List[Region],
+    agent_properties: List[AgentProperties],
+    agent_states: List[AgentState],
+    is_return_map: Optional[bool] = False,
```

Comment:
I think `return_region_index` is a more suitable name for this parameter here.

Reply:
Fair enough. I tend to start boolean parameters with "is" because it typically makes it clear that it is boolean and what question the True or False is answering. This seems clear enough to me though and sounds better.